web_widget_tolocalestring/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
```

Comment:
The following line will be well displayed : 

    ===========================
    "toLocaleString" Web Widget
    ===========================

(same number of char for the title line and the '=' lines)